src/client/components/vision/camera/view_model.ts

```diff
@@ -93,9 +97,40 @@ export class CameraViewModel {
     if (this.model.visualmesh) {
       scene.add(this.visualmesh(this.model.visualmesh))
     }
+    this.model.balls.forEach(ball => scene.add(this.ball(ball)))
```

Comment:
These use Hcw in image, is this going to be a problem if an image hasn't loaded yet?
In theory you should be able to render balls on a blank image